examples/chip-tool/commands/common/CommandInvoker.h

```diff
@@ -234,7 +235,9 @@ CHIP_ERROR InvokeGroupCommand(DeviceProxy * aDevice, void * aContext,
 
     // invoker will be deleted by the onDone call before the return of InvokeGroupCommand
     // invoker should not be used after the InvokeGroupCommand call
-    ReturnErrorOnFailure(invoker->InvokeGroupCommand(aDevice->GetExchangeManager(), groupId, aRequestData));
+    ReturnErrorOnFailure(invoker->InvokeGroupCommand(aDevice->GetExchangeManager(),
+                                                     aDevice->GetSecureSession().Value()->AsSecureSession()->GetFabricIndex(),
```

Comment:
It's worth a comment here explaining that we are assuming `aDevice` has a CASE session already, which is why the `.Value()` and `AsSecureSession()` bits are ok.

Reply:
done